src/test_api.cpp

```diff
@@ -242,6 +242,8 @@ void test_read_write_partial_mat() {
     SUITE_END();
 }
 
+#if 0
+// DEPRECATED: Not used by anyone anymore
```

Comment:
delete?

Reply:
I don't feel strongly about it.
But I like keeping disabled deprecated code in test code for a few revisions, so it is easy to revert back, if we decide it may have been useful, after all.